docs/logging-stdlib-usage.rst

```diff
@@ -0,0 +1,71 @@
+Integration with Python logging module
+--------------------------------------
+
+
+It's possible to tie the Python :mod:`logging` module directly into Google Cloud Logging. To use it,
+create a :class:`CloudLoggingHandler <google.cloud.logging.CloudLoggingHandler>` instance from your
```

Comment:
Instead

```
:class:`~google.cloud.logging.CloudLoggingHandler`
```

----

I realize this was just moved from elsewhere, but "you break it you buy it" or something (more useful to say "if we touch it we can improve it")

I didnt mention it anywhere below, but the prose can be improved (at least
for reviewability) be keeping the lines shorter.